lib/voxpupuli/release/rake_tasks.rb

```diff
@@ -49,3 +50,17 @@
     fail "Unable to find a CHANGELOG.md entry for the #{v} release."
   end
 end
+
+desc 'Check REFERENCE.md.'
+task :check_reference_md do
+  if File.exists?('REFERENCE.md')
+    # the task is defined in the Rakefile in each puppet module
+    # https://github.com/voxpupuli/modulesync_config/blob/a7c683a828332429852a6521598c7ce8652f016a/moduleroot/Rakefile.erb#L40-L44
+    Rake::Task['reference'].invoke
```

Comment:
Perhaps it should check if there's a diff before regenerating and exit before that?

Reply:
good idea, added it